.github/workflows/pypi-publish.yml

```diff
@@ -21,6 +24,20 @@ jobs:
       run: >-
         python -m pip install -U twine wheel
 
+    - name: Create a branch and change the version file for TestPyPI upload
+      if: (github.repository == 'optuna/optuna') && (github.event_name == 'schedule')
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        git config --local user.email gohome.x105.gn@gmail.com
+        git config --local user.name HideakiImamura
```

Comment:
It seems that GitHub user information is required in order to create a branch. I am using my own information. I'm fine with that, but is this okay?

Reply:
I'm not sure if we need to commit the change. I guess the uncommitted changes are also included during the `python setup.py sdist` or `python setup.py bdist_wheel`. If so, we don't need these settings. Have you ever tried it?